src/docs/goose-book/src/getting-started/running.md

```diff
@@ -4,87 +4,42 @@ We will use Cargo to run our example load test application. It's best to get in
- 1: LoadtestTasks        |
-   1:                    | 15.54      | 6          | 136        | 14
-
-=== PER REQUEST METRICS ===
-------------------------------------------------------------------------------
- Name                    | # reqs         | # fails        | req/s  | fail/s
- -----------------------------------------------------------------------------
- GET /                   | 2,240          | 0 (0%)         | 280.0  | 0.000
--------------------------------------------------------------------------------
- Name                    | Avg (ms)   | Min        | Max        | Median
- -----------------------------------------------------------------------------
- GET /                   | 15.30      | 6          | 135        | 14
-
-All 8 users hatched, resetting metrics (disable with --no-reset-metrics).
-
-^C06:03:25 [ WARN] caught ctrl-c, stopping...
-
-=== PER TASK METRICS ===
-------------------------------------------------------------------------------
- Name                    | # times run    | # fails        | task/s | fail/s
- -----------------------------------------------------------------------------
- 1: LoadtestTasks        |
-   1:                    | 2,054          | 0 (0%)         | 410.8  | 0.000
--------------------------------------------------------------------------------
- Name                    | Avg (ms)   | Min        | Max        | Median
- -----------------------------------------------------------------------------
- 1: LoadtestTasks        |
-   1:                    | 20.86      | 7          | 254        | 19
-
-=== PER REQUEST METRICS ===
-------------------------------------------------------------------------------
- Name                    | # reqs         | # fails        | req/s  | fail/s
- -----------------------------------------------------------------------------
- GET /                   | 2,054          | 0 (0%)         | 410.8  | 0.000
--------------------------------------------------------------------------------
- Name                    | Avg (ms)   | Min        | Max        | Median
- -----------------------------------------------------------------------------
- GET /                   | 20.68      | 7          | 254        | 19
--------------------------------------------------------------------------------
- Slowest page load within specified percentile of requests (in ms):
- ------------------------------------------------------------------------------
- Name                    | 50%    | 75%    | 98%    | 99%    | 99.9%  | 99.99%
- -----------------------------------------------------------------------------
- GET /                   | 19     | 21     | 53     | 69     | 250    | 250
+$ cargo run --release -- --host https://drupal-memcache.ddev.site --report-file=report.html
+    Finished release [optimized] target(s) in 0.17s
+     Running `target/release/goose_attack --host 'https://drupal-memcache.ddev.site' --report-file=report.html`
+
+// Some output omitted...
```

Comment:
Go ahead and remove "`// Some output omitted...`". We don't include comments elsewhere.

You could include this line at the end though, as was there before:
```
^C06:03:25 [ WARN] caught ctrl-c, stopping...
```